purchase_order_type_advanced/models/purchase_order.py

```diff
@@ -0,0 +1,31 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
```

Comment:
Please confirm unused imports removal:
```suggestion
from odoo import api, fields, models
```